lib/model/content.dart

```diff
@@ -623,9 +623,13 @@ class LinkNode extends InlineContainerNode {
   }
 }
 
+sealed class MentionNode extends InlineContainerNode {
+  const MentionNode({super.debugHtmlNode, required super.nodes});
+}
+
 enum UserMentionType { user, userGroup }
 
-class UserMentionNode extends InlineContainerNode {
+class UserMentionNode extends MentionNode {
   const UserMentionNode({
     super.debugHtmlNode,
     required super.nodes,
```

Comment:
Hmm, I'm not sure how the comment about silent mentions would apply to this generic `MentionNode`, since it's being introduced for `@topic` mention and not sure how a silent `@topic` would work. I feel this comment is specific to `UserMentionNode`.

Reply:
Oh I see! Yeah that makes sense.

Reply:
A silent @-topic mention might not be super useful, but it does exist:
https://chat.zulip.org/#narrow/channel/7-test-here/topic/mention/near/2007548